test/channels/application_cable/connection_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
+
+class ApplicationCable::ConnectionTest < ActionCable::Connection::TestCase
```

Comment:
- [ ] Please make sure the app has unit tests for all the most important components of the RoR application.